### ISSUE TITLE
error occured after running the project

### ISSUE THREAD

**Issue by leyaannie** [craigtaub/create-react-app-login-form#3]
Failed to load resource: net::ERR_CONNECTION_REFUSED
localhost/:1 Uncaught (in promise) TypeError: Failed to fetch

**Comment by craigtaub** [comment]
Hi @leyaannie 
Do you have an application running on `http://localhost:8080/api/v1/user` ( i.e.`config.USER_URL`)?
That error suggests there is no API-side running. 
This repo helps with the front-end, but im afraid the back-end must be handled seperately. 
It should return a response in the format `{success: true}`

**Comment by leyaannie** [comment]
I just download your project.. I didn't make any changes in it.  In that
zip there is a page config.js which contains this code
export default {
  USER_URL: 'http://localhost:8080/api/v1/user' || process.env.USER_URL
}

**Comment by craigtaub** [comment]
Yep so my project just handles the front-end and assumes you have already got a back-end/API setup. 

If you run this small express app somewhere locally, my project should now work

```
const express = require("express");
const app = express();

app.post("/api/v1/user", (req, res) => { 
    res.json({success: true});
});

const port = 8080;
app.listen(port, () => {
    console.log("Listening on " + port);
});
```

**Comment by leyaannie** [comment]
Ohh... Ok I will try this....   Thank you... 😊

**Comment by craigtaub** [comment]
👍
